web/basics2.textile

```diff
@@ -242,21 +242,21 @@ case classes are used to conveniently store and match on the contents of a class
 scala> case class Calculator(brand: String, model: String)
 defined class Calculator
 
-scala> val hp20b = Calculator("hp", "20b")
-hp20b: Calculator = Calculator(hp,20b)
+scala> val HP20b = Calculator("HP", "20b")
```

Comment:
Ah, sorry I should have mentioned this before: Since this is a class instance and not a enum/type/companion object/etc, can you please keep start these variables uncapitalized?